airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -135,12 +136,12 @@ public AirbyteConnectionStatus checkStorageIamPermissions(final JsonNode config)
           .toList();
 
       if (!missingPermissions.isEmpty()) {
-        LOGGER.error("Please make sure you account has all of these permissions:{}", REQUIRED_PERMISSIONS);
+        LOGGER.warn("Please make sure you account has all of these permissions:{}", REQUIRED_PERMISSIONS);
```

Comment:
Can you add a comment here on why are we doing it this way. The comment can explain that the above approach doesnt work when people use conditional binding for permission handling on their GCS bcuket and that's why we have to do it this way

Reply:
> Can you add a comment here on why are we doing it this way. The comment can explain that the above approach doesnt work when people use conditional binding for permission handling on their GCS bcuket and that's why we have to do it this way

added comment